src/database/language_service.py

```diff
@@ -0,0 +1,41 @@
+from database.models.language import Language
+from models.code_language import LanguageStorable, CodeLanguage
+import database.manager as db_manager
+
+
+class Actions:
+
+    @staticmethod
+    def get_languages_for_repo(repo_id: int) -> [Language]:
+        if isinstance(repo_id, int):
```

Comment:
I'd say remove this isinstance - your type hint is enough.  If someone misuses this it'll crash and then they'll change it.